goose/extractors/title.py

```diff
@@ -46,24 +43,8 @@ def clean_title(self, title):
             pattern = re.compile(self.article.domain, re.IGNORECASE)
             title = pattern.sub("", title).strip()
 
-        # split the title in words
-        # TechCrunch | my wonderfull article
-        # my wonderfull article | TechCrunch
-        title_words = title.split()
-
-        # check if first letter is in TITLE_SPLITTERS
-        # if so remove it
-        if title_words[0] in TITLE_SPLITTERS:
-            title_words.pop(0)
-
-        # check if last letter is in TITLE_SPLITTERS
-        # if so remove it
-        if title_words[-1] in TITLE_SPLITTERS:
-            title_words.pop(-1)
-
-        # rebuild the title
-        title = u" ".join(title_words).strip()
-
+        # Remove Title splitters
+        title = u" ".join(re.split('\|| |:|»|-',title)).strip()
```

Comment:
Ok, If we have "Title | Cool site" on input, we receive "Title   Cool site" in output. I think this double space is not good.